src/async_wrap.cc

```diff
@@ -374,7 +381,7 @@ static void RegisterDestroyHook(const FunctionCallbackInfo<Value>& args) {
   p->target.Reset(isolate, args[0].As<Object>());
   p->propBag.Reset(isolate, args[2].As<Object>());
   p->target.SetWeak(
-    p, AsyncWrap::WeakCallback, v8::WeakCallbackType::kParameter);
+    p, AsyncWrap::WeakCallback, WeakCallbackType::kParameter);
```

Comment:
I think you can now unwrap this into a single line.

Reply:
Do you know what is the max width of line? Linter could break this, no?

Reply:
Rebased and wrapped!